internal/cmd/kafka/command_quota_delete.go

```diff
@@ -53,11 +54,15 @@ func (c *quotaCommand) delete(cmd *cobra.Command, args []string) error {
 	return errs
 }
 
-func (c *quotaCommand) validateArgs(cmd *cobra.Command, args []string) error {
+func (c *quotaCommand) validateArgs(cmd *cobra.Command, args []string) (string, error) {
+	var displayName string
 	describeFunc := func(id string) error {
-		_, err := c.V2Client.DescribeKafkaQuota(id)
+		quota, err := c.V2Client.DescribeKafkaQuota(id)
+		if err == nil && displayName == "" { // store the first valid user name
+			displayName = quota.Spec.GetDisplayName()
+		}
 		return err
 	}
 
-	return deletion.ValidateArgsForDeletion(cmd, args, resource.ClientQuota, describeFunc)
+	return displayName, deletion.ValidateArgsForDeletion(cmd, args, resource.ClientQuota, describeFunc)
```

Comment:
Based on the function name, it seems odd that this returns a string.

Reply:
It is odd, I agree. The problem is that I don't want this clogging up the delete function, but at the same time I don't want to make an extra call just to get the name.

How about this? I rename these functions to `validateArgsAndConfirmDeletion` or something similar and roll the confirmation check into these `validateArgs` functions.